src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java

```diff
@@ -191,6 +202,8 @@ private void load(Path file) throws IOException {
 
     private static final String FORCE_ARTIFACT_VERSION = "force-artifact-version:";
 
+    private static final String REPLACE_ARTIFACT = "replace-artifact:";
```

Comment:
When this option can be used ?

Reply:
RESTEasy changed one artifactId from `org.jboss.resteasy:resteasy-validator-provider-11:*` to `org.jboss.resteasy:resteasy-validator-provider:${version.resteasy}`

Reply:
I see, `replace-artifact` will be a useful feature